editor/src/components/canvas/canvas-utils.ts

```diff
@@ -924,7 +924,7 @@ export function getOriginalFrameInCanvasCoords(
   originalFrames: Array<OriginalCanvasAndLocalFrame>,
   target: ElementPath,
 ): CanvasRectangle | null {
-  for (const originalFrame of originalFrames || []) {
+  for (const originalFrame of originalFrames ?? []) {
```

Comment:
We probably should just null check this, instead of putting in an empty array that we then iterate over.

Reply:
but that's two more lines of code! :D